modules/users/client/config/users.client.routes.js

```diff
@@ -115,6 +101,20 @@
           pageTitle: 'Password reset success'
         }
       })
+      .state('password.forgot', {
```

Comment:
Not critical, but can these state's be added back to where they were before these changes. It's causing an unnecessary diff. This can be misleading when looking back at the history.